e2e/pages/adf/content-services.page.ts

```diff
@@ -613,4 +645,21 @@ export class ContentServicesPage {
         return this.contentList.dataTable.getRow(this.columns.name, rowName);
     }
 
+    async selectFolder(folderName: string): Promise<void> {
+        const folderSelected: ElementFinder = element(by.css(`div[data-automation-id="${folderName}"] .adf-datatable-center-img-ie`));
+        await BrowserVisibility.waitUntilElementIsVisible(folderSelected);
+        await BrowserActions.click(folderSelected);
+    }
+
+    async selectFolderWithCommandKey(folderName: string): Promise<void> {
+        await browser.actions().sendKeys(protractor.Key.COMMAND).perform();
+        await this.selectRow(folderName);
+        await browser.actions().sendKeys(protractor.Key.NULL).perform();
+    }
+
+    async openAndChooseSelectionMode(option: string): Promise<void> {
```

Comment:
can we use the dropdownPage ? please ask @arditdomi about it

Reply:
done.